ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -134,18 +131,18 @@ func extractRayCluster(admissionReview *admissionv1.AdmissionReview) (*ray.RayCl
 	return &rayCluster, nil
 }
 
-func genDNSHostnames(workerGroupSpec ray.WorkerGroupSpec, clusterName string, namespace string, replicaIndex int) (string, error) {
-	numHosts := workerGroupSpec.NumOfHosts
-	if numHosts == 0 {
-		return "", errors.New("workerGroupSpec NumOfHosts not set")
+func genDNSHostnames(numOfHosts int32, groupName string, clusterName string, namespace string, replicaIndex int) (string, error) {
+	if numOfHosts == 0 {
```

Comment:
numOfHosts = 0 is still valid with TPUs though right?

Reply:
I don't think `numOfHosts = 0` is ever valid, `numOfHosts` defaults to 1 and if a user sets it to 0 then no workers would be created per replica.